code/game/atoms_movable.dm

```diff
@@ -423,8 +423,9 @@
 	if(throwing)
 		return 1
 
-	if(!isturf(loc))
-		return 1
+	for(var/x in locs)

```

Comment:
You should still only need the loc check, this is just making sure it isnt inside an object